InitGui.py

```diff
@@ -27,7 +27,7 @@
 class MakerWorkbench (Workbench):
     """Maker Wokbench to create objects"""
     #  Icon in XPM 16x16
-    Icon = "/Resources/icons/Maker_workbench_icon.svg"
+    Icon = "./Resources/icons/Maker_workbench_icon.svg"
```

Comment:
This is the way to properly enunciate the filepath:
`Icon = FreeCAD.getUserAppDataDir()+"Mod"+"/MakerWorkbench/Resources/icons/Maker_workbench_icon.svg"`